subsys/net/lib/lwm2m_client_utils/lwm2m/lwm2m_firmware.c

```diff
@@ -464,41 +613,66 @@ void lwm2m_verify_modem_fw_update(void)
 	sys_reboot(SYS_REBOOT_COLD);
 }
 
-int lwm2m_init_image(void)
+void lwm2m_report_firmware_update_status(uint16_t obj_inst_id)
+{
+	int ret = 0;
+	struct update_counter counter;
+	char path[9+1]; /* '5/[0-65535]/5' -> max len 9 + terminating NULL */
```

Comment:
coud that 9+1 be #define PATH_MAX_LEN 10  or something similar?

Reply:
I'm a bit biased to say but I think that this is better as it's not used anywhere else.

Reply:
This is actually consistent as the same approach is used elsewhere when forming the path. Maybe there should be some common macro-definition for this at the Zephyr-level.

Reply:
Instead of explaining magic numbers in the comment, I'd rather make the code more verbose:
```suggestion
	char path[sizeof("5/65535/5")];
```